src/datasources/auth-api/utils/to-signable-siwe-message.ts

```diff
@@ -0,0 +1,53 @@
+import { SiweMessage } from '@/domain/auth/entities/siwe-message.entity';
+
+/**
+ * The following adheres to the EIP-4361 (SiWe) standard
+ * {@link https://eips.ethereum.org/EIPS/eip-4361}
+ */
+export function toSignableSiweMessage(message: SiweMessage): string {
```

Comment:
Nitpick: Is this intended to be used outside the scope of `SiweApi`? I mean, if it's only used by that class, have you considered to include it there to increase encapsulation, instead of exporting the function globally?

Reply:
I would like to encapsulate but it [will be needed for testing](https://github.com/safe-global/safe-client-gateway/blob/dbb327c929120737eb5b21905400c405c56ab28b/src/routes/auth/auth.controller.spec.ts#L112) the controller.

It is only "used" here (outside of tests) so I think it belongs under the `auth-api`. What do you think?

Reply:
Hmm, then I'd say it makes sense to keep it as it is (globally exported function). To access it in that testing class we would need to export the function in `IAuthApi` and it makes no sense IMO, as it is purely SiWe-related.

Thanks for explaining!